### PR TITLE
loader-utils > 3.0.0 version has removed stringifyRequest method

### PR DESCRIPTION
**Pull request by silentsakky**
https://github.com/webpack/loader-utils/blob/master/CHANGELOG.md#-breaking-changes

### CODE REVIEW

File: src/content/contribute/writing-a-loader.mdx [hunk 1]
```diff
@@ -241,7 +241,12 @@ export default function loader(source) {
 
 ### Absolute Paths
 
-Don't insert absolute paths into the module code as they break hashing when the root for the project is moved. There's a [`stringifyRequest`](https://github.com/webpack/loader-utils#stringifyrequest) method in `loader-utils` which can be used to convert an absolute path to a relative one.
+Don't insert absolute paths into the module code as they break hashing when the root for the project is moved. You can use below code to convert absolute paths to relative one.
```
Comment (chenxsan) [inline src/content/contribute/writing-a-loader.mdx:244]:
```suggestion
Don't insert absolute paths into the module code as they break hashing when the root for the project is moved. You can use below code to convert absolute paths to relative ones.
```

File: src/content/contribute/writing-a-loader.mdx [hunk 2]
```diff
@@ -241,7 +241,12 @@ export default function loader(source) {
...
+
+```js
+// loaderContext is this inside loader function
```
Comment (chenxsan) [inline src/content/contribute/writing-a-loader.mdx:247]:
```suggestion
// `loaderContext` is same as `this` inside loader function
```

File: src/content/contribute/writing-a-loader.mdx (hunk 1 above)
Comment (silentsakky) [reply src/content/contribute/writing-a-loader.mdx]: Fixed

File: src/content/contribute/writing-a-loader.mdx (hunk 2 above)
Comment (silentsakky) [reply src/content/contribute/writing-a-loader.mdx]: Fixed

**Review by chenxsan — APPROVED** [review]
Thanks.

**Pull request merged**